specification/sql/resource-manager/Microsoft.Sql/stable/2018-06-01/managedInstances.json

```diff
@@ -0,0 +1,213 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-06-01",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/sqlAgent/current": {
+      "get": {
+        "tags": [
+          "ManagedInstances"
+        ],
+        "description": "Gets current instance sql agent configuration.",
+        "operationId": "SqlAgentConfiguration_Get",
```

Comment:
Uri says the resource type is `SqlAgent`, operation id says the resource type is `SqlAgentConfiguration`. These must be aligned. Please change one or the other.

Reply:
I have changed it to SqlAgent